lib/versions.js

```diff
@@ -15,23 +15,28 @@ export const versions = {
   wpCli: "2.4.0",
 }
 
-export async function checkLisaVersion() {
-  let localVersion = await asyncExec(
-    `npm show version@triggerfishab/lisa-cli version`
-  )
-  let npmVersion = await asyncExec(
-    `npm show version@triggerfishab/lisa-cli dist-tags.latest`
+const getVersions = async () => {
+  let versionsOutput = await asyncExec(
+    `npm outdated @triggerfishab/lisa-cli --json --global --all || true`
   )
+  return JSON.parse(versionsOutput.stdout.trim())
+}
 
-  localVersion = localVersion.stdout.trim()
-  npmVersion = npmVersion.stdout.trim()
+export async function checkLisaVersion() {
+  const parsedOutput = await getVersions()
+  if (Object.keys(parsedOutput).length === 0) {
+    return
+  }
 
-  if (semver.compare(localVersion, npmVersion) === -1) {
+  const versions = parsedOutput["@triggerfishab/lisa-cli"]
+  const currentVersion = versions?.current
+  const latestVersion = versions?.latest
```

Comment:
namnen är lite mög, men ville få upp nått så ni kan se.

the gist `npm outdated @triggerfishab/lisa-cli --json --global --all || true`

ger tebax en json som är antingen 
* tomt object om det inte finns några uppdateringar eller
* ett objekt med paket som behöver/kan uppdateras.